config/default.cfg

```diff
@@ -22,7 +22,7 @@ cfg$title <- "default"
 cfg$regionmapping <- "config/regionmappingH12.csv"
 
 #### Current input data revision (<mainrevision>.<subrevision>) ####
-cfg$revision <- 5.964
+cfg$revision <- 5.971
```

Comment:
wow this looks like a big jump

Reply:
there were many errors in mrremind, and the in-between broken caches are not to be used.